gpt2.py

```diff
@@ -35,11 +38,9 @@ def text_generator(model_name, prefix, temperature, **kwargs):
                          batch_size=5
                          )
 
-def _random_prefix():
-    """
-    prefix random generator
-    """
-    pass
+data = pd.read_csv(file_name)
+corpus = data['Fortune Cookie Quotes']
+start_word = _random_prefix(corpus)
```

Comment:
Would `seed_word` be an accurate description?

Reply:
yes. 
This reminds me of adding a seed option to set a fix seed for reproducibility.